src/get_maintainer_wrapper.sh

```diff
@@ -114,12 +142,28 @@ function execute_get_maintainer()
     return 1
   fi
 
-  say $SEPARATOR
-  say "HERE:"
   cd $kernel_root
-  eval perl $script $options "$FILE_OR_DIR"
+  local -r script_output="$(eval perl $script $script_options "$FILE_OR_DIR")"
   cd $original_working_dir
 
+  say $SEPARATOR
```

Comment:
Add double-quote.

For a reference, take a look at:

https://github.com/koalaman/shellcheck/wiki/Sc2086